plugins/modules/ip_share.py

```diff
@@ -25,9 +25,6 @@
 from linode_api4.objects import Instance
 
 ip_share_spec = {
-    # Disable the default values
-    "label": SpecField(type=FieldType.string, required=False, doc_hide=True),
-    "state": SpecField(type=FieldType.string, required=False, doc_hide=True),
```

Comment:
It looks like state is actually used in this module contrary to this snippet, so it might make sense to bring it back as a normal field 👍

Reply:
Nice catch! Just fixed it and verified locally